### DISCUSSION TITLE
KFR integration into Yocto

### DISCUSSION THREAD

**Discussion by NyzhnykAI** [kfrlib/kfr#223] [General]
Hello
Trying to add KFR library to Yocto based image. Installation step seems passing but dependent module can not find the KFR with `find_package`. Did I missed anything? Does anything specific should be added to the KFR build configuration?
KFR 6.0.2 was used for build.

**Comment by dancasarin** [comment]
Hello,
Could you show the cmake command line? Are there KFRConfig* files at `CMAKE_INSTALL_PREFIX/lib/cmake/kfr` ?

**Reply by NyzhnykAI** [reply]
The only custom flag added to the cmake configuration is `EXTRA_OECMAKE += "-DKFR_ARCH=sse2"`. In installation folder I can see only `lib/cmake/kfr/KFRConfigVersion.cmake`
